pre_commit/commands/run.py

```diff
@@ -248,6 +253,15 @@ def _all_filenames(args: argparse.Namespace) -> Collection[str]:
         return git.get_staged_files()
 
 
+def _has_diff(
+        diff: Optional[Tuple[int, bytes, Optional[bytes]]],
+) -> bool:
+    if diff is not None:
+        return diff[0] == 1
```

Comment:
oh hmmm, I don't think we need to keep track of the returncode at all, that seems to be an accidental implementation detail of the current code